src/simple/strategy.cpp

```diff
@@ -75,10 +77,10 @@ void Strategy::update(const common::MarketData& market_data) {
               roq::Side::Sell,
               _quantity,
               best.bid_price);
-        if (FLAGS_real_trading)
-          at(0).sell_ioc(_quantity, best.bid_price);
-        else
-          LOG(WARNING) << "Trading is *not* enabled (use --real-trading)";
+        // if (FLAGS_real_trading)
```

Comment:
Quick question;
What's the motivation for dropping this flag?

The reason for it's existence is that the user shouldn't be able to start a strategy without explicitly (and consciously) choosing to actually trade. It's safer to get an error reminding you that you have to think: "Do I really want to trade?", rather than suddenly sending a flood of orders by mistake.

Otherwise I'm happy to pull this request.